assets/admin/editor-wizard/limited-text-control.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+import { TextareaControl, TextControl } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Text control with input limited by a given maximum length.
+ *
+ * @param {Object}   props           Component properties.
+ * @param {string}   props.label     Label for the field.
+ * @param {string}   props.value     Value for the field.
+ * @param {Function} props.onChange  Callback for when the value changes.
+ * @param {number}   props.maxLength Maximum length for the field.
+ * @param {boolean}  props.multiline Whether if multiline input (textarea) must be used or not.
+ */
+const LimitedTextControl = ( {
+	label,
+	value,
+	onChange,
+	maxLength,
+	multiline = false,
+} ) => {
+	const changeValueIfAllowed = ( newValue ) => {
```

Comment:
Quickly checking the `TextareaControl` and `TextControl`, I think they send the extra props we send to them to the `input` and `textarea`. Maybe we could simplify this by adding a `maxlength` prop to the `<Control>`?

Reply:
Much better! 👏  Thanks!